crates/holochain/src/core/state/element_buf.rs

```diff
@@ -160,6 +160,14 @@ where
         Ok(self.headers.get(header_address)?.map(Into::into))
     }
 
+    pub fn get_header_with_reader<'r, 'a: 'r, R: Readable>(
```

Comment:
If we need to create a bunch of these `_with_reader` versions we should just convert the underlying store to Used instead of Fresh, but this is fine. Fresh stores are kind of a convenience and a way to prevent massive refactoring, but I think Used is preferable in general.

Reply:
Yeh it turned out I didn't really need this but I thought I'd leave it in in case we did later but also happy to take it out